src/solver/opamCudf.ml

```diff
@@ -968,7 +1045,11 @@ let extract_explanations packages cudfnv2opam reasons : explanation list =
         | `Missing (_, sdeps', fdeps') -> sdeps = sdeps' && fdeps = fdeps'
         | _ -> false)
   in
+  log ~level:3 "Explanations: %a" Pp_explanation.pp_explanationlist explanations;
   match explanations with
+  | [] ->
+    OpamConsole.error_and_exit `Internal_error
+      "Internal error while computing conflict explanations: sorry, please report."
```

Comment:
This wants to be more explicit, especially if something were to happen where a lot of users suddenly see it. The best thing is probably to direct users to an/the issue page, which can then be closed as and when we manage to eliminate the problem!

Reply:
(adding comments from https://github.com/ocaml/opam/pull/4982#discussion_r779385255)

Reply:
Is it better now?

Reply:
x-ref https://github.com/ocaml/opam/pull/4981#discussion_r789162066